web/src/ResourceStatusSummary.tsx

```diff
@@ -299,12 +299,18 @@ function statusCounts(statuses: ResourceStatus[]): StatusCounts {
   }
 }
 
-function ResourceMetadata(props: { counts: StatusCounts }) {
+function ResourceMetadata(props: {
+  counts: StatusCounts
+  tiltConnected?: boolean
+}) {
   let { totalEnabled, healthy, pending, unhealthy } = props.counts
   useEffect(() => {
     let favicon: any = document.head.querySelector("#favicon")
     let faviconHref = ""
-    if (unhealthy > 0) {
+    if (props.tiltConnected === false) {
+      document.title = `… disconnected ┊ Tilt`
+      faviconHref = "/static/ico/favicon-gray.ico"
```

Comment:
we might have to put this at a public url? since you can't guarantee we'll be able to load this if we're disconnected

Reply:
i'm open to suggestions here! for most users, i'd expect that the favicons would be cached.

Reply:
you can throw them in https://github.com/tilt-dev/tilt.build/tree/master/src/assets and it will show up in https://tilt.dev/assets/svg/blog.svg

Reply:
great! that's [#1089](https://github.com/tilt-dev/tilt.build/pull/1089) and when that's merged in, i'll update this pr with the url.

Reply:
done!